spec/tasks/mongoid_slug_rake_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+require 'rake'
+
+describe 'mongoid_slug:set' do
+  before :all do
+    load File.expand_path('../../../lib/tasks/mongoid_slug.rake', __FILE__)
+    Rake::Task.define_task(:environment)
+  end
+
+  context 'when models parameter is passed' do
+    let!(:uninitalized_slug1) { UninitalizedSlugFirst.create(name: 'uninitalized_slug1') }
+    let!(:uninitalized_slug2) { UninitalizedSlugSecond.create(name: 'uninitalized_slug2') }
+
+    it 'Goes though all documents of passed models and sets slug if not already set' do
+      class UninitalizedSlugFirst
```

Comment:
@dblock I didn't redefine, I reopened these classes to include  `Mongoid::Slug`, as I wanted to simulate situation where few objects of class are already created and `Mongoid::Slug` is included later. In such case slug do not initialize for previously created objects and running task will set slugs for such objects.
Correct me, if this is not proper way.